model/src/main/proto/interaction_object.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Object type of the interaction object
+enum ObjectType {
+  NORMALIZED_STRING = 0;
+  SIGNED_INT = 1;
+  NON_NEGATIVE_INT = 2;
+  REAL = 3;
+  BOOLEAN = 4;
+  NUMBER_WITH_UNITS = 5;
+  SET_OF_HTML_STRING = 6;
+}
+
+// Structure for any interaction object
+message InteractionObject {
+  ObjectType object_type = 1;
+  oneof value {
+    string normalized_string = 2;
+    int32 signed_int = 3;
+    uint32 non_negative_int = 4;
+    float real = 5;
+    bool bool_value = 6;
+    NumberWithUnits number_with_units = 7;
+    // repeated fields cannot be within a oneof so lists are
+    // wrapped into this parameter.
+    List set_of_html_string = 8;
+  }
+}
+
+// Structure containing lists. Note we should fill only
```

Comment:
Maybe instead we can just make this a StringList. At that point, the extra caveat in the documentation doesn't seem necessary since it's probably sufficiently clear that the purpose of the proto is to be a StringList.

Reply:
Done